client/my-sites/plans/plan-overview/plan-features/feature.jsx

```diff
@@ -0,0 +1,45 @@
+/**
```

Comment:
I like it this idea but in that case we should define standalone components. Right now it's not the case since we're defining the `.plan-feature__button` class in the parent component's stylesheet.